tembo-operator/src/snapshots/volumesnapshots.rs

```diff
@@ -271,14 +326,6 @@ fn generate_volume_snapshot(
             error!("VolumeSnapshotContent name is empty for instance {}", name);
             Action::requeue(tokio::time::Duration::from_secs(300))
         })?;
-    let volume_snapshot_class_name = snapshot_content
-        .spec
-        .volume_snapshot_class_name
-        .as_ref()
-        .ok_or_else(|| {
-            error!("VolumeSnapshotClass name is empty for instance {}", name);
-            Action::requeue(tokio::time::Duration::from_secs(300))
-        })?;
```

Comment:
This is removed as it's not needed in the `VolumeSnapshot` and will be populated and assumed from the `VolumeSnapshotContents`